docs/resource-sharing/os-backfill-containers.md

```diff
@@ -213,6 +213,26 @@ the `--memory` argument tells Docker to kill the container if its total memory u
     Additionally, the processes that manage jobs also use some amount of memory.
     Therefore, it is important to give the container some extra room.
 
+
+### Advanced: Advertising additional pilot attributes
+
+You can put arbitrary additional attributes in the machine ads that you advertise to the OSPool.
+These attributes will show up when you do `condor_status -l` on your machine.
```

Comment:
```suggestion
These attributes will show up when users run `condor_status -l` against your glidein.
```

I think it's safe to assume that some of these sites don't have HTCondor client tools